docs/openapi/open-api.yml

```diff
@@ -69,3 +138,68 @@ components:
         - mSettings
         - mStandings
         - mTeam
+        - players_wl
+        - proTeamSchedules_wl
+        - mDraftDetail
+    
+    TransactionPayload:
+      description: Payload for POST request body to change a lineup
+      type: "object"
+      required:
+        - isLeagueManager
+        - teamId
+        - type
+        - memberId
+        - scoringPeriodId
+        - executionType
+        - items
+      properties:
+        isLeagueManager: 
+          description: not sure if this can be boolean and not sure what happens if its true
+          type: string
```

Comment:
Is it actually a string not a boolean?

Reply:
just tried boolean and both work, I will change it to boolean